### PR TITLE
interactive overlay

### LINKED ISSUE #1185: Overlay

### PR DESCRIPTION
**Pull request by anna-wro**
screen shot 2017-08-01 at 16 26 09
Closes #1185

### CODE REVIEW

File: src/components/overlay/pages/overlay-interactive.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import DocsActiveBlock from 'components/DocsActiveBlock';
+import Overlay from '../Overlay';
+
+const Overlays = () => {
+  const settings = [];
+
+  return <div>
+    <DocsActiveBlock settings={settings}>
+      <div>
+        <Overlay/>
```
Comment (kdzwinel) [inline src/components/overlay/pages/overlay-interactive.jsx]: huh, that's how we use overlay? I'd make more sense to make it wrap content it's covering IMO. I'd make it more flexible (you can cover only part of page). @anna-wro @matikrk @aju WDYT?

Comment (anna-wro) [inline src/components/overlay/pages/overlay-interactive.jsx]: we used it this way in the style guide, I even checked with much older versions (70-80)

Comment (anna-wro) [inline src/components/overlay/pages/overlay-interactive.jsx]: I checked in other projects and we also use it in the way you described (wrapping content).

**Comment by anna-wro** [comment]
@kdzwinel I updated the example

**Comment by anna-wro** [comment]
screen shot 2017-08-07 at 08 09 28

**Comment by kdzwinel** [comment]
@matikrk @aju WDYT about making overlay `absolute` by default and having `fixed` as an option? This way we could use overlay to cover only some part of the page (e.g. payment iframe, everything w/o the header, etc.), not only to always cover the whole page.

**Comment by aju** [comment]
@kdzwinel I'm ok with it. It would just require some refactoring

**Comment by anna-wro** [comment]
@aju, @kdzwinel - I did as you suggested
screen shot 2017-08-08 at 08 50 52

**Comment by anna-wro** [comment]
screen shot 2017-08-08 at 11 19 32

Changed default positioning - it's fixed now, you can change it to absolute by checking the 'partial' prop

**Pull request merged**